glass/shells.py

```diff
@@ -708,15 +708,15 @@ def partition_restrict(
     return part
 
 
-def redshift_grid(
+def _uniform_grid(
     zmin: float,
     zmax: float,
     *,
     dz: float | None = None,
     num: int | None = None,
```

Comment:
Maybe something more generic here?

```suggestion
    start: float,
    stop: float,
    *,
    step: float | None = None,
    num: int | None = None,
```

Reply:
Good idea